app/src/tx_reviewer.rs

```diff
@@ -0,0 +1,594 @@
+        } = self.prepare_output(output, current_index, temp_data)?;
+        let review_message = self.get_str_from_range(review_message)?;
+        let alph_amount = self.get_str_from_range(alph_amount)?;
+        let address = self.get_str_from_range(address)?;
+        let address_field = Field {
+            name: "Address",
+            value: address,
+        };
+        let alph_amount_field = Field {
+            name: "ALPH",
+            value: alph_amount,
+        };
+        if token.is_none() {
+            let fields = [address_field, alph_amount_field];
+            review(&fields, review_message)?;
+            self.reset();
+            return Ok(());
+        }
+
+        let TokenIndexes {
+            token_id,
+            token_amount,
+        } = token.unwrap();
+        let token_id = self.get_str_from_range(token_id)?;
+        let token_amount = self.get_str_from_range(token_amount)?;
+        let fields = [
+            address_field,
+            alph_amount_field,
+            Field {
+                name: "Token ID",
+                value: token_id,
+            },
+            Field {
+                name: "Token Amount",
+                value: token_amount,
+            },
+        ];
+        review(&fields, review_message)?;
+        self.reset();
+        Ok(())
+    }
+
+    pub fn review_tx_details(
+        &mut self,
+        unsigned_tx: &UnsignedTx,
+        path: &[u32],
+        temp_data: &NVMBuffer<'static, NVM_DATA_SIZE>,
+    ) -> Result<(), ErrorCode> {
+        match unsigned_tx {
+            UnsignedTx::NetworkId(byte) => Self::review_network(byte.0),
+            UnsignedTx::GasAmount(amount) => Self::review_gas_amount(amount),
+            UnsignedTx::GasPrice(amount) => self.review_gas_price(amount),
+            UnsignedTx::Inputs(inputs) => {
+                if let Some(current_input) = inputs.get_current_item() {
+                    self.review_input(
+                        current_input,
+                        inputs.current_index as usize,
+                        inputs.size(),
+                        path,
+                        temp_data.read()?,
+                    )
+                } else {
+                    Ok(())
+                }
+            }
+            UnsignedTx::FixedOutputs(outputs) => {
+                if let Some(current_output) = outputs.get_current_item() {
+                    self.review_output(
+                        current_output,
+                        outputs.current_index as usize,
+                        temp_data.read()?,
+                    )
+                } else {
+                    Ok(())
+                }
+            }
+            _ => Ok(()),
+        }
+    }
+
+    pub fn review_tx_id(tx_id: &[u8; 32]) -> Result<(), ErrorCode> {
+        let hex: [u8; 64] = utils::to_hex(&tx_id[..]).unwrap();
+        let hex_str = bytes_to_string(&hex)?;
+        let fields = [Field {
+            name: "Tx ID",
```

Comment:
Let's use `Transaction ID`

Reply:
It has been fixed in this PR: https://github.com/alephium/ledger-alephium/pull/17